public/index.html

```diff
@@ -2,12 +2,12 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <meta property="og:url"           content="%PUBLIC_URL%" />
+    <meta property="og:url"           content="%PUBLIC_URL%/" />
     <meta property="og:type"          content="website" />
     <meta property="og:title"         content="Dialogue COVID-19 Info" />
     <meta property="og:description"   content="Dialogue Chloe, an automated medical assistant, provides Canadians with personalized access to the latest localized public health information as the government's response to the COVID-19 outbreak continues to develop." />
     <meta property="og:locale"        content="en" />
-    <meta property="og:image"         content="%PUBLIC_URL%/fb-share-image.jpg" />
+    <meta property="og:image"         content="%PUBLIC_URL%/share-image.jpg" />
```

Comment:
if this is just for those IG things, and since it might mess with deployment to other domains, I'd consider adding a new environment variable: `DEPLOYMENT_URL`

which can then be injected the same way by CRA:
```suggestion
    <meta property="og:image"         content="%DEPLOYMENT_URL%/share-image.jpg" />
```

Reply:
Minimizing the change set. We should rework the cci config beyond this